### PR TITLE
Bootcamp dev

### PR DESCRIPTION
**Pull request by FREDVUNI**
# Ticket No
- https://trello.com/c/lW3WXqYC
- https://trello.com/c/94zi5wpb
Screenshot (112)

# Summary of changes
- Changed the navigation menu title from **angie** to **Project Zoe** and also menu item **Chat** to **Connect**

# How should this be tested? [Screenshots, Video or Type instructions]
- screenshot

# Notes for reviewers
- Notes for reviewers here

# Out of scope
-  Out of scope here

### CODE REVIEW

File: src/modules/login/Login.tsx [hunk 1]
```diff
@@ -32,10 +36,12 @@ function Login() {
         history.push(localRoutes.profile);
       },
       () => {
+        console.log("whyy")
```
Comment (jacksono) [inline src/modules/login/Login.tsx]: This console log should be removed.

File: src/modules/login/Login.tsx [hunk 2]
```diff
@@ -23,7 +23,11 @@ function Login() {
   const dispatch = useDispatch();
   const history = useHistory();
   const onSubmit = (data: any, actions: FormikHelpers<any>) => {
-    post(
+    // dispatch(handleLogin({}));
+    Toast.success('Authentication success');
+    history.push(localRoutes.profile);
+    if ((1 + 3) == 2) {
+      post(
```
Comment (jacksono) [inline src/modules/login/Login.tsx]: There should be no changes here

File: src/index.tsx [hunk 3]
```diff
@@ -1,37 +1,37 @@
...
+//   beforeSend: (event) => {
+//     if (
+//     // Exclude "localhost" envs from issue tracker
+//       window.location.hostname === 'localhost'
+//     ) {
+//       return null;
+//     }
+//     return event;
+//   },
+// });
```
Comment (jacksono) [inline src/index.tsx]: This should not be commented out

File: src/index.tsx [hunk 4]
```diff
@@ -1,37 +1,37 @@
...
+import React from 'react'; 
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import { ThemeProvider } from '@material-ui/styles';
 import CssBaseline from '@material-ui/core/CssBaseline';
 import 'react-toastify/dist/ReactToastify.css';
-import * as Sentry from '@sentry/react';
-import { Integrations } from '@sentry/tracing';
+// import * as Sentry from '@sentry/react';
+// import { Integrations } from '@sentry/tracing';
```
Comment (jacksono) [inline src/index.tsx]: This should not be commented out

File: src/components/layout/NavMenu.tsx [hunk 5]
```diff
@@ -195,7 +197,8 @@ const NavMenu = (props: any) => {
         alignContent="center"
         justify="center"
       >
-        <img src={appLogo} alt="logo" className={classes.logo} />
+        {/* <img src={appLogo} alt="logo" className={classes.logo} /> */}
```
Comment (jacksono) [inline src/components/layout/NavMenu.tsx]: It should be safe to delete this line since we are replacing it with the line below it.

File: src/components/layout/NavMenu.tsx [hunk 6]
```diff
@@ -56,7 +56,7 @@ const routes: IAppRoute[] = [
     icon: TodayIcon,
   },
   {
-    name: 'Chat',
+    name: 'Connect',
```
Comment (jacksono) [inline src/components/layout/NavMenu.tsx:59]: 👍🏾

File: package.json [hunk 7]
```diff
@@ -18,6 +18,7 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.54",
     "@material-ui/pickers": "^3.2.10",
+    "@material-ui/styles": "^4.11.5",
```
Comment (jacksono) [inline package.json]: Why did you need to add this package?

File: .eslintignore [hunk 8]
```diff
@@ -4,3 +4,4 @@ my-app*
 packages/react-scripts/template
 packages/react-scripts/fixtures
 fixtures/
+src/*
```
Comment (jacksono) [inline .eslintignore]: This folder should not be ignored

**Review by jacksono — CHANGES_REQUESTED** [review]
@FREDVUNI  I have left a few comments that need to be addressed.

**Comment by FREDVUNI** [comment]
Alright. This is probably why my client side is currently not working as it
should

File: package.json (hunk 7 above)
Comment (FREDVUNI) [reply package.json]: There was an error raised by material-ui and I found out that installing styles would solve the issue. I'm going to uninstall it though because we have since made quite a number of changes

**Review by jacksono — APPROVED** [review]
LGTM

**Comment by FREDVUNI** [comment]
# Ticket No
- https://trello.com/c/00zJRFve

Screenshot (118)

Screenshot (114)

Screenshot (116)

# Summary of changes
- Edit person details through pop-up modal in place within the Persons Table
- Closing modal dialog returns to the people page when the URL has parameter showEdit

# How should this be tested? [Screenshots, Video or Type instructions]
- screenshot


# Notes for reviewers
- Notes for reviewers here


# Out of scope
-  Out of scope here

**Comment by FREDVUNI** [comment]
# Ticket No
- https://trello.com/c/DCDA6rAH

Screenshot (120)

# Summary of changes
- Removed coming soon text and moved the send email button

# How should this be tested? [Screenshots, Video or Type instructions]
- screenshot


# Notes for reviewers
- Notes for reviewers here


# Out of scope
-  Out of scope here

**Comment by FREDVUNI** [comment]
# Ticket No
- https://trello.com/c/tUAwrgWp

# Summary of changes
- Changed the landing page from profile to dashboard

# How should this be tested? 
- Demo


# Notes for reviewers
- Notes for reviewers here


# Out of scope
-  Out of scope here

**Comment by Roland-Sankara** [comment]
LGTM
